doc/any.xml

```diff
@@ -574,6 +562,378 @@ public:
+              contained value if instance is non-empty, otherwise
+              <code>typeid(void)</code>.</simpara></returns>
+
+              <notes><simpara>Useful for querying against types known
+              either at compile time or only at
+              runtime.</simpara></notes>
+            </method>
+          </method-group>
+        </class>
+
+        <function name="swap">
+          <type>void</type>
+          <parameter name="lhs">
+            <paramtype><classname>basic_any</classname> &amp;</paramtype>
+          </parameter>
+          <parameter name="rhs">
+              <paramtype><classname>basic_any</classname> &amp;</paramtype>
+          </parameter>
+
+            <effects><simpara>Exchange of the contents of
+            <code>lhs</code> and
+            <code>rhs</code>.</simpara></effects>
+
+            <throws><simpara>Nothing.</simpara></throws>
+        </function>
+
+        <overloaded-function name="any_cast">
+          <signature>
+            <template>
+              <template-type-parameter name="T"/>
+            </template>
+
+            <type>T</type>
+
+            <parameter name="operand">
+              <paramtype><classname>basic_any</classname> &amp;</paramtype>
+            </parameter>
+          </signature>
+
+          <signature>
+            <template>
+              <template-type-parameter name="T"/>
+            </template>
+
+            <type>T</type>
+
+            <parameter name="operand">
+              <paramtype><classname>basic_any</classname> &amp;&amp;</paramtype>
+            </parameter>
+          </signature>
+
+          <signature>
+            <template>
+              <template-type-parameter name="T"/>
+            </template>
+
+            <type>T</type>
+
+            <parameter name="operand">
+              <paramtype>const <classname>basic_any</classname> &amp;</paramtype>
+            </parameter>
+          </signature>
+
+          <signature>
+            <template>
+              <template-type-parameter name="ValueType"/>
+            </template>
+
+            <type>const ValueType *</type>
+
+            <parameter name="operand">
+              <paramtype>const <classname>basic_any</classname> *</paramtype>
+            </parameter>
+          </signature>
+
+          <signature>
+            <template>
+              <template-type-parameter name="ValueType"/>
+            </template>
+
+            <type>ValueType *</type>
+
+            <parameter name="operand">
+              <paramtype><classname>basic_any</classname> *</paramtype>
+            </parameter>
+          </signature>
+
+          <purpose><simpara>Custom keyword cast for extracting a value
+          of a given type from an
+          <code><classname>basic_any</classname></code>. Also awailable from <code>boost</code> namespace.</simpara></purpose>
+
+          <returns><simpara> If passed a pointer, it returns a
```

Comment:
Suggestion. I would reformulate to something like that.

First part: "**For overloads with pointer to ``basic_any``:** return a similarly qualified pointer..." _return_ is intentionally without **s** at the end because we have multiple overloads

Second part: "**For overloads with reference to ``basic_any``:** If T is ValueType, **return** a copy of the held value, otherwise, if T is a reference **return** a reference to the held value"

Reply:
Those are the copy-pasted words of the original library author. I'd rather leave them as is.